mkdocs.yml

```diff
@@ -1,9 +1,15 @@
+---
```

Comment:
Apparently, the yaml syntax defines that every file must start with the 3 dashes. Otherwise, the yamllinter wouldn't have told me to add them, I guess.

Reply:
weird specification... but ok, fair enough :)